packages/transaction-controller/src/EtherscanRemoteTransactionSource.ts

```diff
@@ -40,7 +40,7 @@ export class EtherscanRemoteTransactionSource
     this.#isTokenRequestPending = false;
   }
 
-  isSupportedNetwork(chainId: Hex, _networkId: string): boolean {
+  isSupportedNetwork(chainId: Hex): boolean {
```

Comment:
whatttt this method just took the networkId and did nothing with it previously? lol

Reply:
yeah, it's because it was required in the `RemoteTransactionSource` interface 🙃